### PR TITLE
EPIC SEE-3105 Extend SSE tools configuration controls

### PR DESCRIPTION
**Pull request by BorisP-GDS**
EPIC SEE-3105 Extend SSE tools configuration controls

### CODE REVIEW

File: express/src/views/clients/client-details.njk [hunk 1]
```diff
@@ -88,6 +150,44 @@
...
+        actions: {
+          items: [{
+            href: urls.changeBackChannelLogoutUri,
+            text: "Change",
+            visuallyHiddenText: "Back channel logout URI"
+          }]
+        }
+      }, 
+      {
+        key: {text: "Sector identfier URI"},
```
Comment (MartinH-DI) [inline express/src/views/clients/client-details.njk:175]: Should be: identifier

File: express/src/views/clients/client-details.njk [hunk 2]
```diff
@@ -88,6 +150,44 @@
...
+        value: {html: sectorIdentifierUriContainer},
+        actions: {
+          items: [{
+            href: urls.changeSectorIdentifierUri,
+            text: "Change",
+            visuallyHiddenText: "Sector identfier URI"
```
Comment (MartinH-DI) [inline express/src/views/clients/client-details.njk:181]: Should be: identifier

File: express/src/views/clients/client-details.njk (hunk 1 above)
Comment (BorisP-GDS) [reply express/src/views/clients/client-details.njk]: Fixed

File: express/src/views/clients/client-details.njk (hunk 2 above)
Comment (BorisP-GDS) [reply express/src/views/clients/client-details.njk]: Fixed

**Review by MartinH-DI — APPROVED** [review]
This all looks good

**Review by Ryan-Andrews99 — APPROVED** [review]

**Pull request merged**